src/test/java/seedu/fridgefriend/command/AddCommandTest.java

```diff
@@ -0,0 +1,42 @@
+package seedu.fridgefriend.command;
+
+import org.junit.jupiter.api.Test;
+import seedu.fridgefriend.exception.InvalidDateException;
+import seedu.fridgefriend.exception.InvalidInputException;
+import seedu.fridgefriend.food.ExpiryDate;
+import seedu.fridgefriend.food.FoodCategory;
+import seedu.fridgefriend.food.FoodStorageLocation;
+import seedu.fridgefriend.food.Fridge;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+class AddCommandTest {
+
+    /**
```

Comment:
I am not sure if we require comments for these two methods, the details of the test is already in the method name. It might make the comments unnecessary.